website/datataking.html

```diff
@@ -0,0 +1,293 @@
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+<!--my css code. -->
+    <link rel="stylesheet" href="index.css" type="text/css">
+
+  </head>
+<!--I am also having a hard time spreading the nav bar equally spaced (maybe use an id)
+I also can't get the drop down menu to work-->
+<header id="mainmenu">
+  <nav class="navbar navbar-expand-lg nav-justified" style="background:black;">
+    <div>
+  <a class="navbar-brand text-white" href="../index.html"> </a> </div>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse justify-content-center nav-fill" id="navbarSupportedContent">
+    <ul class="nav navbar-nav nav-fill" class= ".active">
+      <li class="active">
+        <a class="nav-link text-white" href="../index.html">Home<span class="sr-only">(current)</span></a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link text-white" href="about.html">About</a></li>
+      <li class="nav-item">
+          <a class="nav-link text-white" href="materials.html">Materials</a> </li>         <li class="nav-item dropdown">
+       <a class="nav-link dropdown-toggle text-white" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Tutorials
+          </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item text-black" href="hardware.html">Telescope Design</a>
+            <a class="dropdown-item text-black" href="software.html">Software Setup</a>
+            <a class="dropdown-item text-black" href="datataking.html">Taking data</a>
+            <a class="dropdown-item text-black" href="analysis.html">Analysis</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="faq.html">FAQ</a> </li>
+        <li class="nav-item">
+          <a class="nav-link text-white" href="contact.html">Contact</a> </li>
+        </ul>
+</div>
+</div>
+</nav>
+<h2 id="centerheader" style= "border:solid;border-color:black;" font-family:"courier new", courier, arial; >Taking Data</h2>
+</header>
+
+    <body>
+
+  <div>
+    <ul class="sidenav" style="text-align:left;color:white;border-bottom-left-radius:5px;border-bottom-right-radius:5px;">
+  <li> <a href="#firstdata">Take Your First Radio Data</a> </li>
+  <li>  <a href="#spectrum">Create a Spectrum Flowchart and Save the Data</a> </li>
+  <li> <a href="#datasize">Is our Data Size Correct?</a> </li>
+  <li> <a href="#package">Data taking with CHART installed package</a> </li>
+  </ul>
+  </div>
+
+  <div id="top_pad"></div>
```

Comment:
This div is being covered up by the side menu (div on lines 61-68). Not sure why, but maybe compare with the other pages?